README.md

```diff
@@ -200,12 +204,22 @@ For a full list of contributors refer to the [authors page](./AUTHORS.md)
 
 ## Issues
 
+### Usage Questions
+
+Pose questions and help answer them on [Stack
+Overflow](https://stackoverflow.com/questions/tagged/fairlearn) with the tag
+`fairlearn` or on [Gitter](https://gitter.im/fairlearn/community#).
+
 ### Regular (non-security) issues
 
-Please submit a report through [GitHub issues](https://github.com/fairlearn/fairlearn/issues). A maintainer will respond promptly as follows:
+Issues are meant for bugs & feature requests. Please submit a report through
+[GitHub issues](https://github.com/fairlearn/fairlearn/issues). A maintainer
+will respond promptly as follows:
 - **bug**: triage as `bug` and provide estimated timeline based on severity
-- **feature request**: triage as `feature request` and provide estimated timeline
-- **question** or **discussion**: triage as `question` and either respond or notify/identify a suitable expert to respond
+- **feature request**: triage as `feature request` and provide estimated
+  timeline
+- **question** or **discussion**: triage as `question` and either respond or
+  notify/identify a suitable expert to respond
```

Comment:
Isn't it confusing to have a "question" listed here given that we want to direct questions to Stack Overflow and Gitter? Can we be a bit more specific here?

Reply:
Well spotted! Thank you.